src/commands/Games/coinflip.ts

```diff
@@ -0,0 +1,56 @@
+import { CommandStore, KlasaMessage } from 'klasa';
+import { SkyraCommand } from '../../lib/structures/SkyraCommand';
+import { UserSettings } from '../../lib/types/settings/UserSettings';
+import { MessageEmbed } from 'discord.js';
+import { getColor, cleanMentions } from '../../lib/util/util';
+
+const enum CoinTypes { Heads, Tails }
+export default class extends SkyraCommand {
+
+	private readonly cdnTypes = ['heads', 'tails'] as const;
```

Comment:
Nit: rename to coin types and you're not using it

Reply:
I am, in the thumbnail. Also, coinTypes is the localized version of that, its named cdnTypes since I only use it there

Reply:
Didn't see that, tha8!